backend/lib/models/User.js

```diff
@@ -1,33 +1,31 @@
-const { model, Schema } = require("mongoose");
+const { Schema, model } = require("mongoose");
+const { schema: locationSchema } = require("./Location");
+const { isValidEmail } = require("../utils");
 
-const UserSchema = new Schema(
+const userSchema = new Schema(
   {
-    _id: {
+    about: { maxLength: 100, trim: true, type: String },
+    authId: { required: true, type: String },
```

Comment:
@joaofnds Is this supposed to be the auth0 id? If so shouldn't it only be on IndividualUser?

Reply:
@joshmorel Not according to the [data model](https://github.com/FightPandemics/DataModel/blob/master/v2/scripts/JSON_Schema/applicationDB/users.json#L23), but maybe we should revisit that! What do you think @Naraujo13? 

Having the `authID` in the User model would allow the organization and its owner to log in independently

Reply:
@joaofnds  But where is the `authId` coming from on organization creation? https://app.zeplin.io/project/5ebb0c7bc826984e0abb929b/screen/5ebf499e21d04848a62753dc

We're only creating an Auth0 account based on email/password at user sign up time no?

Is my understanding correct, that you sign up as an individual user. You can then post as an individual user or create an organization and post as that, where the `ownerId` associates the org with the signed-in individual (if we want multiple users per org - that would be a post-MVP feature).

Reply:
Yeah, you're right, I'll remove it! We can add it later if needed anyway 😄

Great catch btw!